integration-tests/cxf-soap/src/main/resources/application.properties

```diff
@@ -64,5 +56,4 @@ camel.beans.codeFirstServiceEndpoint.address=/codefirst
 camel.beans.codeFirstServiceEndpoint.features[0]=#bean:loggingFeature
```

Comment:
Not something that we have to do for this PR, but I think we should avoid `camel.beans`. IIRC it works via reflection at runtime. It'd be more optimized to have CDI producers for those items.

Reply:
Thanks, good point, let me do it.